.github/workflows/nix.yml

```diff
@@ -0,0 +1,28 @@
+name: LNbits CI / nix
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    paths:
+      - 'flake.nix'
+      - 'flake.lock'
+      - 'pyproject.toml'
+      - 'poetry.lock'
+  pull_request:
+    paths:
```

Comment:
If I understand correctly:
- the flow will only run if one of these `path` files is changed
- it will not run when the 'lnbits' logic is changed
- it only checks that the new dependency or new versions of dependencies are nixable

Suggestion:
 - call the file `check-nix-dependencies.yml`

Reply:
> If I understand correctly:

Yes

> call the file check-nix-dependencies.yml

I prefer to keep the filename, because in the future this workflow file can contain more nix specific stuff